app/utils/convertInventoryToGeoJson.ts

```diff
@@ -34,15 +33,16 @@ export default async function getGeoJsonData(inventoryData: any) {
       return {
         type: 'Feature',
         properties: {
-          isPolygonComplete: onePolygon.isPolygonComplete,
           ...metadata,
         },
         geometry: {
-          type: 'LineString',
-          coordinates: onePolygon.coordinates.map((oneCoordinate: any) => [
-            oneCoordinate.longitude,
-            oneCoordinate.latitude,
-          ]),
+          type: 'Polygon',
+          coordinates: [
+            onePolygon.coordinates.map((oneCoordinate: any) => [
+              oneCoordinate.longitude,
+              oneCoordinate.latitude,
```

Comment:
You wrote longitude before latitude here in contrast to all other places where latitude is listed first. Is this an error?

Reply:
Fixed it myself.

Reply:
I was testing geoJSON on this website [https://geojson.io](https://geojson.io) and found out that for point [lat, long] is correct and for Polygon it should be [long, lat]

Reply:
You can check by exporting you geoJSON

Reply:
I think there's some other problem, I'm looking into this

Reply:
From my tests with a single tree and multiple tree export on this website, we seem to need [longitude, latitude] for single point as well as polygons, e.g.
```
  "geometry": {
    "coordinates": [
      9.935418322225985,
      53.547927406691315
    ],
    "type": "Point"
  },
```
```
      "geometry": {
        "type": "Polygon",
        "coordinates": [
          [
            [
              9.9353538196508,
              53.54860199113149
            ],
            [
              9.93557510185127,
              53.54860039762337
            ],
            [
              9.935572419770125,
              53.54870716926696
            ],
            [
              9.935352478590488,
              53.548703982105074
            ],
            [
              9.9353538196508,
              53.54860199113149
            ]
          ]
        ]
      }
```

But I am irritated looking at above API example why our device location defined it differently.
```
  "deviceLocation": {
    "coordinates": [
      9.935418322225985,
      53.547927406691315
    ],
    "type": "Point"
  },
```

We should really check this thoroughly :-)

Reply:
Yes, I'm debugging the issue and so converted this PR to Draft.